src/Instances/Ref.lua

```diff
@@ -0,0 +1,14 @@
+--!strict
+
+--[[
+	The symbol used to denote the reference of an instance when working with the
+	`New` function.
+]]
+
+local Package = script.Parent.Parent
+local PubTypes = require(Package.PubTypes)
+
+return {
+	type = "Symbol",
+	name = "RefKey"
```

Comment:
To be consistent with `Children`, `name` should be `Ref`.

Reply:
Thanks for the catch, this got passed the unit testing since it had to do with type checking. I'll work on addressing that now.

Reply:
I pushed a fix for this issue, but I'm leaving this comment unresolved so you can verify this fix properly address your comment.